src/router/index.js

```diff
@@ -53,35 +53,35 @@ const routes = [
     name: 'Register',
     component: Register,
     meta: {
-      requiresNonLogged: true
-    }
+      requiresNonLogged: true,
```

Comment:
For readability reasons I would recommend renaming this to
`requiresAuth: true/false`